Aptoide/bds-sdk
# [APPC-1601] Added new rules to cafe bazaar redirects and binds

**What does this PR do?**

  Adds new rules to cafe bazaar redirects and bind

**Database changed?**

No

**Where should the reviewer start?**

CafeBazaarUtils.java
InstallDialogActivity.java
WalletUtils.java

**How should this be manually tested?**

Change language to Persian (translate on google to see which one it is on the phone languages)
Press install from the install dialog and notification and check if it redirects to cafe bazaar url (with no Cafe Bazaar store)
Check if the code is the same in both android-appcoins-billing and ads module.
The second use case (bind cafe bazaar wallet is not possible to test since they don't the wallet released yet.)
 
**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1601

**Code Review Checklist**
